src/utils.h

```diff
@@ -13,4 +14,13 @@
 
 #define DBG() fprintf(stderr, "%s:%d\n", __func__, __LINE__)
 
+#define UNUSED __attribute__((unused))
+
```

Comment:
Let's use casting to `void` for this purpose instead

Reply:
fixed.